dependencies.yaml

```
@@ -140,10 +169,13 @@ dependencies:
           - cupy>=12.0.0
       - output_types: requirements
         packages:
-          - cupy-cuda11x>=12.0.0
+          - &cupy_pip cupy-cuda11x>=12.0.0
+      - output_types: pyproject
+        packages:
+          - *cupy_pip
```

See previous suggestion.
```suggestion
          - cupy-cuda11x>=12.0.0
```

---

fixed